core_screenshot_config.yaml

```diff
@@ -41,7 +41,14 @@ primary:
   MT:
     overseerScript: page.manualWait(); await page.waitForDelay(30000); page.done();
     message: waiting 30 sec to load MT
-
+  
+  ND:
+    overseerScript >
```

Comment:
Hi there! Just a quick note that you need a colon here like `overseerScript: >`, otherwise it breaks the YAML. Going to push a fix